components/SideNavigation.tsx

```diff
@@ -7,6 +7,7 @@ import type { FC } from 'react';
 
 type SideNavigationProps = {
   navigationKey: NavigationKeys;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
i don't know what this should be typed as yet

Reply:
Sadly, the `context` is legacy and complex and will be removed in the future. For now this typing is fine